_posts/2020-11-xx-azure-devops-pipeline-as-code.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title: Upgrade Your Classic Pipelines to Pipeline as Code in Azure DevOps
+author: lassiautio
+excerpt: >
+  Azure DevOps' pipeline as code first sounds a bit scary for those who are familiar with older classic pipeline GUI.
+  This blog post will go through the basics of how to create a build pipeline as code in YAML and gives some tips on where to get help to write YAML pipelines.
+  If you're not yet familiar with YAML pipelines, I consider reading this and try them.
+tags:
+ - Azure DevOps
+ - DevOps
+ - continuous integration
+ - Build pipeline
+ - CI
+---
+
+Azure DevOps used to have only graphical build pipelines, known as Classic Pipeline GUI, which were edited only in the web portal.
+In Build 2019 Microsoft released pipeline as code, where pipelines are defined with YAML (aka YAML pipeline).
+In this blog post, I will go through the basics of YAML pipelines and compare them with the classic pipelines.
```

Comment:
This is the part where we typically try to "sell" the blog: Make readers interested in investing a bit of time to read it. This tells what the blog is about, which is good, but would it be possible to find more impact by telling what could be achieved by it? Even one sentence explaining why pipelines as code are cool, or what is the effect if you start using this?